test/index.ts

```diff
@@ -182,42 +190,358 @@ describe('.getV3Filename()', function() {
 })
 
 describe('.toV3()', function() {
-  const salt = Buffer.from(
-    'dc9e4a98886738bd8aae134a1f89aaa5a502c3fbd10e336136d4d5fe47448ad6',
-    'hex',
-  )
-  const iv = Buffer.from('cecacd85e9cb89788b5aab2f93361233', 'hex')
-  const uuid = Buffer.from('7e59dc028d42d09db29aa8a0f862cc81', 'hex')
-
-  it('should work with PBKDF2', function() {
+  const pw = 'testtest'
+  const salt = 'dc9e4a98886738bd8aae134a1f89aaa5a502c3fbd10e336136d4d5fe47448ad6'
+  const iv = 'cecacd85e9cb89788b5aab2f93361233'
+  const uuid = '7e59dc028d42d09db29aa8a0f862cc81'
+
+  const strKdfOptions = { iv, salt, uuid }
+  const buffKdfOptions = {
+    salt: Buffer.from(salt, 'hex'),
+    iv: Buffer.from(iv, 'hex'),
+    uuid: Buffer.from(uuid, 'hex'),
+  }
+
+  const makePermutations = (...objs: Array<object>): Array<object> => {
```

Comment:
Can you add some comments to clarify this part?

Reply:
Comments added, let me know if additional clarity is needed.